os_faults/drivers/tcpcloud.py

```diff
@@ -29,7 +29,7 @@ class TCPCloudNodeCollection(node_collection.NodeCollection):
 
     def connect(self, network_name):
         LOG.info("Connect network '%s' on nodes: %s", network_name, self)
-        task = {'osa_network_mgmt': {
+        task = {'fuel_network_mgmt': {
```

Comment:
osa_network_mgmt file doesn't exist in os_faults.ansible.modules only fuel_network_mgmt.py does.  so I changed it back to using this and it looks (so far) like it will work with our deployment